### DISCUSSION TITLE
WDPhotTools package - running the photometric fit code

### DISCUSSION THREAD

**Discussion by cm775** [cylammarco/WDPhotTools#13] [Q&A] [answered]
Hi,

I’m trying to run the photometric fitting part of thr WDPhotTools package, but I’m running into an issue. In the documentation an example fit is run using ftr.fit(….) but I can’t find anything written in the document about what I need to define as ftr. I don’t know what the right part of the package to import/define is in order to run the photometric fitting. Any help would be greatly appreciated.

**Comment by cylammarco** [comment, marked as answer]
Hi @cm775 , sorry about that, I have added a couple of lines in the tutorial:

```
from WDPhotTools.fitter import WDfitter
ftr = WDfitter()

ftr.fit(....)

```
It is reflected in [https://wdphottools.readthedocs.io/en/latest/tutorials/fitting.html](https://wdphottools.readthedocs.io/en/latest/tutorials/fitting.html)

Please do a force refresh to clear the cache, from experience Readthedocs uses a pretty aggressive cache...

**Reply by cm775** [reply]
Hi, Thanks for getting back to me so quickly. I'm trying to exactly replicate your fitting example, but I'm getting the following errors when running the ftr.show_corner_plot part: "ValueError: cannot reshape array of size 4 into shape (3,3)" and "ValueError: Provided figure has 4 axes, but data has dimensions K=3". Is there anything I need to change from the example to have this run? Also wanted to ask how to pull out the values of the independent variables (Teff & logg) separately from the plots.

**Reply by cylammarco** [reply]
Hi @cm775 , I have accidentally included the distance in the `truths` kwarg in the README.md example when calling the `show_corner_plot()`. It is now corrected. It should read:

```
ftr.show_corner_plot(
    figsize=(10, 10),
    display=True,
    kwarg={
        "quantiles": [0.158655, 0.5, 0.841345],
        "show_titles": True,
        "truths": [3550, 7.45],
    },
)

```
